### PR TITLE
Tests: podman will no longer utilize UID entries

### PR DESCRIPTION
**Pull request by aborah-sudo**
https://bugzilla.redhat.com/show_bug.cgi?id=2093311
https://bugzilla.redhat.com/show_bug.cgi?id=2109410

### CODE REVIEW

File: multihost_test/Bugzillas/test_bz_automation.py [hunk 1]
```diff
@@ -48,3 +48,18 @@ def test_crashes(self, multihost):
                            "grep test_user "
                            "/tmp/newroot/etc/passwd").stdout_text
         execute_cmd(multihost, "rm -fr /tmp/newroot")
+
+    def test_2093311(self, multihost, create_backup, create_localuser):
+        """
+        :title: Podman will no longer utilize UID entries
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_bz_automation.py:54]: The title seems confusing. I would change it for "sub*id files can also contain IDs"

File: multihost_test/Bugzillas/test_bz_automation.py [hunk 2]
```diff
@@ -48,3 +48,18 @@ def test_crashes(self, multihost):
...
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2093311
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2109410
+        """
+        execute_cmd(multihost, "yum install -y  podman")
+        user_uid = execute_cmd(multihost, "id local_anuj").stdout_text.split(" ")[0].split('(')[0].split("=")[1]
+        execute_cmd(multihost, f"sed -i -e 's/local_anuj/{user_uid}/g' /etc/subuid")
+        execute_cmd(multihost, f"sed -i -e 's/local_anuj/{user_uid}/g' /etc/subgid")
+        result = execute_cmd(multihost, 'su - local_anuj -c "podman run fedora cat /proc/self/uid_map"').stdout_text
+        for i in execute_cmd(multihost, f"grep {user_uid} /etc/subuid").stdout_text[:-1].split(":"):
+            assert i in result.split()
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_bz_automation.py:65]: I'm having problems understanding this part. Can you explain it?

Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_bz_automation.py:65]: As we have replaced the user_name(local_anuj) with uid in /etc/subuid and /etcsubgid, so here i am searching the f"user_id" like (grep {user_uid} /etc/subuid) and spiting it  and matching it with variable "result".


result = (su - local_anuj -c "podman run fedora cat /proc/self/uid_map")
result =  [ 0 , 1004, 1, 362144, 65536 ] 

user_uid = execute_cmd(multihost, "id local_anuj").stdout_text.split(" ")[0].split('(')[0].split("=")[1] = 1004
execute_cmd(multihost, f"grep {user_uid} /etc/subuid").stdout_text[:-1].split(":") =  [1004, 362144, 65536]

Now i am making sure that all elements  [1004, 362144, 65536] present in result

File: multihost_test/Bugzillas/test_bz_automation.py (hunk 1 above)
Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_bz_automation.py]: Done

File: multihost_test/Bugzillas/test_bz_automation.py (hunk 2 above)
Comment (ikerexxe) [reply multihost_test/Bugzillas/test_bz_automation.py:65]: Thanks for the explanation

**Review by ikerexxe — APPROVED** [review]
LGTM

**Pull request merged**